gradle/module-versioning.gradle

```diff
@@ -0,0 +1,101 @@
+
+task bumpVersions(type: BumpVersionTask)
```

Comment:
```suggestion
/**
 * This task should be used to easily bump the major/minor/patch version of a fabric-api module.
 * It will automatically bump the versions of dependent modules.
 */
task bumpVersions(type: BumpVersionTask)
```